CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.0] - TBD
```

Comment:
I believe we are going to bump this and all the other ones to 2.0.0 since we have some potentially breaking changes here

Reply:
I've made a PR to update the package version and changelog in the 2.0.0 release branch: https://github.com/SpringRoll/Springroll-Seed/pull/111

I noticed there was a slight difference between the changelog on `main` and the one in the templates. This change will keep it all consistent, and it just needs to be pulled into the template branches.